srcpkgs/jmol/template

```diff
@@ -0,0 +1,28 @@
+# Template for 'jmol'
+pkgname=jmol
+version=14.29.17
+revision=1
+noarch=yes
+maintainer="Brenton Horne <brentonhorne77@gmail.com>"
+homepage="http://jmol.sourceforge.net/"
+depends="virtual?java-environment"
+license="LGPL-3"
+hostmakedepends="unzip"
```

Comment:
hostmakedepends= should be directly above depends=

Reply:
Shall do, but might I ask why hasn't this been built into xlint as a check? After all I ran xlint quite a few times while writing this file to make sure my file would meet all the criteria to be added to this repository and if it told me about this issue I would have fixed it. 

Not an attack, I admire this project, especially how yas handled the recent issue with xtraeme's vanishing act, just might be an idea to implement. :)